hoover/search/views.py

```diff
@@ -122,3 +122,40 @@ def whoami(request):
             'logout': reverse('logout') + '?next=/',
         },
     })
+
+@csrf_exempt
+@limit_user
+def batch(request):
+    t0 = time()
+    body = json.loads(request.body.decode('utf-8'))
+    collections = collections_acl(request.user, body['collections'])
+    queries = body.get('queries')
+    aggs = body.get('aggs')
+
+    if not collections:
+        return JsonResponse({'status': 'error', 'reason': "No collections selected."})
+    if not queries:
+        return JsonResponse({'status': 'error', 'reason': "No items to be searched."})
+    if len(queries) > 100:
+        return JsonResponse({'status': 'error', 'reason': "Too many queries. Limit is 100."})
+
+    success = False
+    try:
+        res = es.batch_count(
+            queries,
+            collections,
+            aggs
+        )
+        res['status'] = 'ok'
+        return JsonResponse(res)
+
+    except es.SearchError as e:
+        return JsonResponse({'status': 'error', 'reason': e.reason})
+
+    finally:
+        signals.search.send('hoover.batch', **{
```

Comment:
Let's create a different signal, `batch`, so we can log it separately. And let's log the number of queries too.